meow/eme/common.py

```diff
@@ -79,9 +88,10 @@ def compute_interface_s_matrix(
         U, s, V = np.linalg.svd(S)
         S = np.diag(s) @ U @ V
 
-    # ensure reciprocity:
+    # ensure reciprocity: HACK?
     if enforce_reciprocity:
-        S = 0.5 * (S + S.T)
+        S = 0.5 * (S + S.T.conj())
```

Comment:
I don't think this change is correct. Reciprocity implies a symmetric S-matrix, not a complex-symmetric one. That said... I think reciprocity is only valuable for guided modes which are all nicely orthogonal. Since we're seeing a lot of spurious modes I'm not sure if enforcing reciprocity is the right call and hence I don't mind changing the default flag to False.

I think they tackle that issue in this paper https://arxiv.org/pdf/1301.2458.pdf but I don't have enough time/brainpower right now to read through it, haha

Reply:
Jep, I forgot, that I changed that (and didn't change it back), because I think one should not use enforce reciprocity anyways :)

Reply:
(as the interface matrices do not need to be symmetric, from what I see)

Reply:
I think the crux to this is in the C matrix in the paper, but I would need to have a closer look...

Reply:
indeed